src/monopoly.js

```diff
@@ -134,6 +140,41 @@ export default class Monopoly {
     return this;
   }
 
+  loadPropertyArray(){
+    this.propertyArray = JSON.parse(fs.readFileSync('properties.JSON'));
+    this.propertyArray.forEach((property) => {
+      switch (property.functionID) {
+        case 1:
```

Comment:
Also, an alternative to the switch statement is using  map pointing to the functions:

```
const PROPERTY_FUNCTIONS = {
    'standard': standardProperty,
    'railroad': railroad,
    'utility': utility,
    // etc. 
};
```

Then in here:
```
property.landOnFunction = PROPERTY_FUNCTIONS[property.type];
```